lib/uploader.ts

```diff
@@ -163,34 +171,194 @@ export class Uploader {
 		this._notifiers.push(notifier)
 	}
 
+	/**
+	 * Notify listeners of the upload completion
+	 * @param upload The upload that finished
+	 */
+	private _notifyAll(upload: Upload): void {
+		for (const notifier of this._notifiers) {
+			try {
+				notifier(upload)
+			} catch (error) {
+				// ignore
```

Comment:
Shouldn't we at least log?

Reply:
This was just refactoring, as the same code was already used in two places (chunked upload + normal upload) and would have been duplicated a third time.
So I did not change it, but of course I can add the logging here :)